src/parser.cpp

```diff
@@ -36,4 +37,121 @@ types::VPDMapVariant Parser::parse()
     return l_parser->parse();
 }
 
+int Parser::updateVpdKeyword(const types::WriteVpdParams& i_paramsToWriteData)
+{
+    auto [l_fruPath, l_inventoryObjPath, l_redundantFruPath] =
```

Comment:
no point in getting all these till "writeKeywordOnHardware" succeeds. 
Do that operation first?